xep-0198.xml

```diff
@@ -409,6 +425,13 @@
 </failed>
 ]]></example>
   <p>Stream management errors SHOULD be considered recoverable; however, misuse of stream management MAY result in termination of the stream.</p>
+  <p>When a remote entity acknowledges that it has handled a number of stanzas that is higher than the amount of stanzas that it was sent (by sending an 'h' value that is to high), the local entity SHOULD generate a stream error and close the stream:</p>
+  <example caption='Server closes old stream that was replaced by new stream'><![CDATA[
```

Comment:
Is the caption text right? Shouldn't it be something like "Closing a stream because a remote entity reported a impossible number of handled stanzas".

Reply:
I believe I messed up a copy/paste somewhere. I'll correct it.